src/lib.rs

```diff
@@ -252,4 +287,26 @@ mod tests {
         // assert_eq!(blurhash, "LFL4=pI8%foujXofRkWC.loyH?V{");
         assert_eq!(blurhash, "LNAdAqj[00aymkj[TKay9}ay-Sj[");
     }
+
+    #[test]
+    #[cfg(feature = "image")]
+    fn test_jelly_beans_image() {
+        let img = image::open("data/octocat.png").unwrap();
+
+        let blurhash = encode_image(4, 3, &img.to_rgba8()).unwrap();
+
+        assert_eq!(blurhash, "LNAdAqj[00aymkj[TKay9}ay-Sj[");
+    }
+
+    #[test]
+    #[cfg(feature = "image")]
```

Comment:
```suggestion
```

Since `image` is a dev-dep, this can go

Reply:
No, as the `encode_image`/`decode_image` only exist with the `image` feature flag.

Reply:
The test already has the `image`-dependency, but not the `image`-flag.

Reply:
Ooooh wow. That's slightly yuck, but I'm with you now.